### ISSUE TITLE
Working model of Self-driving car using Convolutional Neural Network, Raspberry Pi and Arduino

### ISSUE THREAD

**Issue by wrryu09** [GDSC-Ewha-4th/Study-paperReview#40]
# Working model of Self-driving car using Convolutional Neural Network, Raspberry Pi and Arduino

`Proceedings of the 2nd International conference on Electronics, Communication and Aerospace Technology (ICECA 2018) IEEE Conference Record # 42487; IEEE Xplore ISBN:978-1-5386-0965-1`

🔶 This paper proposes a working model of self-driving car which is capable of driving from one location to the other or to say on different types of tracks such as curved tracks, straight tracks and straight followed by curved tracks.

Image were resized, converted into grayscale and fed into the Convolutional Neural Network. The output of Neural Network helps in taking the directional decision. Once it is done, the controller simple sends the corresponding signal and the car moves in that particular direction.

---

**network configuration**

image resized to 320 x 230

Convolutional Neural Network : 128 input nodes, 2 hidden layers of 32 nodes each and output layer consisting of 4 nodes for each of the 4 output.

dropout of 0.5 to avoid overlearning

input & hidden layers : ‘Relu’ activation function used

Output layer : ‘Softmax’ activation function ised

Batch size : 10

number of epoch : 3

---

![image](https://user-images.githubusercontent.com/98469609/200158706-7d89ee7d-311b-45a4-a5fa-e8e7fb94b687.png)
🔹 h/w used : Raspberry Pi, Pi Camera, Arduino Microcontroller

🔹 s/w used : Arduino IDE, OpenCV and Spyder environment, Raspberry Pi Cam Interface

🔶 Result

With the help of Image Processing and Machine Learning a successful model was developed

If the scope and reliability improved, it would make the overall design risk-free from accidents.

**Comment by minha62** [comment]
CNN을 사용해서 track에서 차가 어떤 방향으로 가야 할지 예측하는 논문이군요! 
좋은 내용 감사합니다:)
